tests/functional/test_task_list.py

```diff
@@ -1,5 +1,16 @@
+import responses
+
+
 def test_task_list_success(run_line, load_api_fixtures):
     load_api_fixtures("task_list.yaml")
     result = run_line("globus task list")
     assert "SUCCEEDED" in result.output
     assert "TRANSFER" in result.output
+    # check that empty filters aren't passed through
+    filters = dict(
+        x.split(":") for x in responses.calls[0].request.params["filter"].split("/")
+    )
+    if "completion_time" in filters:
+        assert filters["completion_time"] == "", filters
+    if "request_time" in filters:
+        assert filters["request_time"] == "", filters
```

Comment:
We know in the unadorned `task list` case that nothing is passed. The fact that this is conditional is looking a little odd to me.

Can we simplify it down to this?
```suggestion
assert "completion_time" not in filters
assert "request_time" not in filters
```

If not, I just want to know why.

Reply:
Yeah, this was maybe excessively test-driven, I just wanted to start with something that would fail. Fixed.